File: src/LIT.Smabu.UseCases/Catalogs/UpdateGroup/UpdateCatalogGroupCommand.cs
﻿using LIT.Smabu.Domain.CatalogAggregate;
using LIT.Smabu.UseCases.Shared;

namespace LIT.Smabu.UseCases.Catalogs.UpdateGroup
{
    public record UpdateCatalogGroupCommand(CatalogId CatalogId, CatalogGroupId Id, string Name, string Description) : ICommand
    {

    }
}

File: src/lit.smabu.react/src/types/domain/update-catalog-group-command.ts
import { CatalogGroupId } from './catalog-group-id';
import { CatalogId } from './catalog-id';
export interface UpdateCatalogGroupCommand {
  catalogId?: CatalogId;
  catalogGroupId?: CatalogGroupId;
  name?: string | null;
  description?: string | null;
}
